api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -3238,4 +3239,38 @@ public void mergePatients_shouldMoveAllObsWithSameHierarchy() throws Exception {
 		assertEquals(2, encounterService.getEncounter(57).getObsAtTopLevel(true).size());
 	}
 
+	/**
+	 * @see PatientServiceImpl#getDuplicatePatientsByAttributes(List) 
+	 */
+	@Test(expected = APIException.class)
```

Comment:
Can you junit rules? So that you check the error message too otherwise it could be another failure since APIExceptions are thrown allover the code base